lib/containers/runtime.pm

```diff
@@ -0,0 +1,77 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test Docker’s btrfs storage driver features for image and container management
+# Maintainer: qac team <qa-c@suse.de>
+package containers::runtime;
+use Mojo::Base qw(consoletest);
+use testapi;
+use Test::Assert ':all';
+
+sub new{
+    my ($class, $runtime) = @_;
+    my $self = {
+	runtime => $runtime,
+    };
+    bless $self, $class;
+    return $self;
+}
```

Comment:
You are using `Mojo::Base` and there is a more modern perl way, see https://docs.mojolicious.org/Mojo/Base

```suggestion
has runtime => undef;
```

Reply:
Thank you for this. fixed.